src/style/colors.ts

```diff
@@ -162,7 +163,9 @@ export class Colors {
     return validColors ? undefined : results[0];
   }
 
-  getColorTint(colorValue: string | OpaqueColorValue, tintKey: string | number) {
+  getColorTint(colorValue: string | OpaqueColorValue,
+    tintKey: string | number,
+    options: ColorTintOptions = {shouldReverseOnDark: true}) {
```

Comment:
Try to declare options in a way that works with your default logic
for example: instead of calling the option `shouldReverseOnDark` and set its default to true, maybe have an option called `avoidReverseOnDark`, this way its falsy default value will work with your logic. 

Why is it important? cause if you will add more options in the future, let's call it `anotherOption` and the user will only pass that option like this `getColorTint('#efefef', {anotherOption: true})` then your `shouldReverseOnDark` option will become undefined/false and the user won't understand why the default behavior has changed.  

sorry for the long explanation, let me know if it's not clear...